holmes.py

```diff
@@ -230,6 +233,16 @@ def alertmanager(
 
     try:
         issues = source.fetch_issues()
+        if alertmanager_label:
+            label_parts = alertmanager_label.split('=')
+            if len(label_parts) != 2:
+                logging.error(f"The label {alertmanager_label} is of the wrong format use '--alertmanager-label key=value'")
+                return
+            alert_label_key, alert_label_value = label_parts
+            issues = [issue for issue in issues if issue.raw and issue.raw.get("labels", {}).get(alert_label_key, None) == alert_label_value]
```

Comment:
Is it possible for `issue.raw` to not be defined? If it isn't possible, don't check for it.

Reply:
it could in theory be None correct? and then None.get might throw an exception